metrics/perplexity/test_perplexity.py

```diff
@@ -0,0 +1,36 @@
+import datasets
+from datasets import load_dataset
+
+
+def main():
```

Comment:
Can you format this file as a `pytest` script ?

To do so:
- you just have to remove the `if name == "main":\n main()` at the end, and rename the `main()` function to something like `test_perplexity()`
- remove the print statements, you can use comments or logging if you want
- use `assert` statements or `numpy.testing.assert_allclose` to make the the resulting values are the correct ones (you can hardcode the expected values directly in the script)

Then you should be able to do
```
pytest metrics/perplexity
```

Reply:
I was mainly using this file to make the examples, and was leaning towards deleting it before merging. Do you think it would be useful to keep it? Or is it ok for me to get rid of it?

Reply:
Oh ok I thought it was a test script sorry. You can remove it yes !